internal/constants.go

```diff
@@ -18,7 +18,9 @@ const (
 	BootTimeoutSecondsDefault = 30
 	BootRetrySecondsDefault   = 1
 	ConfigFileName            = "configuration.toml"
+	ConfigStemApp             = "edgex/appservices/"
 	ConfigStemCore            = "edgex/core/"
+	ConfigStemDevice          = "edgex/devices/"
```

Comment:
Add TODO to move these in to the constants file in go-mod-core-contracts where all the common constants reside.

Reply:
add the TODO comments. will create a new issue there